src/SprykerEco/Zed/ArvatoRss/Business/Api/Converter/RiskCheckResponseConverter.php

```diff
@@ -28,17 +28,21 @@ public function convert(stdClass $response)
         $responseTransfer->setResultText($response->Decision->ResultText);
 
         if (isset($response->Details)) {
-            $responseTransfer->setBillingAddressValidation(
-                $this->convertAddressValidationResponse(
-                    $response->Details->BillingCustomerResult->ServiceResults->AddressValidationResponse
-                )
-            );
+            if (isset($response->Details->BillingCustomerResult->ServiceResults->AddressValidationResponse)) {
```

Comment:
Pleas split this part into functions, maybe adding fast return.

Reply:
Functionality splitted. Tests added.